main.py

```diff
@@ -9,19 +9,16 @@
     NoviActionPushInt,
     NoviActionSendReport,
     NoviActionSetBfdData,
+    NoviActionAddIntMetadata as OFNoviActionAddIntMetadata,
```

Comment:
@viniarck I believe those classes should be imported from `napps.amlight.noviflow.pyof.v0x0.action` instead of `napps.amlight.noviflow.of_core.v0x0.action`, no?

Reply:
@italovalcy, good observation, Italo. I think you're right, comparing with `of_core` it should be from `pyof` instances since these will inherent from `ActionExperimenter(ActionHeader)`, I've just pushed this new commit fixing it [3bf7088](https://github.com/kytos-ng/noviflow/pull/19/commits/3bf7088d387f7ebb0255b091f7ab59a8b644313c)